src/Select/Select.interface.ts

```diff
@@ -59,6 +60,7 @@ export interface SelectProps extends WithLabel<SelectInnerProps> {}
 
 export interface SelectState {
   isOpened: boolean
+  isFocused: boolean
```

Comment:
`shouldn't it be `isHovered` ?

Reply:
I first started to implement it along with `isFocused` but I ended up noticing that I didn't need it.

Reply:
`isFocused` allows to track whether or not you clicked to open the component.
If you have a better name, I'll take it.

Reply:
Oh I see, I misunderstood what you meant. `isFocused` is alright